bazel-mode.el

```diff
@@ -107,6 +107,11 @@
   (add-hook 'before-save-hook #'bazel-mode--buildifier-before-save-hook
             nil :local))
 
+;;;###autoload
+(add-to-list 'auto-mode-alist '("BUILD\\'" . bazel-mode))
+(add-to-list 'auto-mode-alist '("WORKSPACE\\'" . bazel-mode))
```

Comment:
This will only put the first form in the autoload file. You have to repeat the autoload cookie before every form.
However, it might be better and faster to add only one entry, such as

```lisp
(cons (rx (or "BUILD" "WORKSPACE" ".bazel") eos) #'bazel-mode)
```